src/presenter/trip-presenter.js

```diff
@@ -33,6 +36,12 @@ export default class TripPresenter {
   }
 
   #renderPoints() {
+
+    if (this.#points.length < 1) {
```

Comment:
Все же стандартная практика сравнивать с 0
```suggestion
    if (this.#points.length === 0) {
```